src/Log/Writer.php

```diff
@@ -250,9 +250,9 @@ protected function triggerLogEvent($level, $message, array $context = array())
 		// If the event dispatcher is set, we will pass along the parameters to the
 		// log listeners. These are useful for building profilers or other tools
 		// that aggregate all of the log messages for a given "request" cycle.
-		if (isset($this->dispatcher))
+		if ($this->dispatcher instanceof DispatcherInterface)
 		{
-			$this->dispatcher->trigger('system.onLog', array($level, $message, $context));
+			$this->dispatcher->trigger('onLog', array($level, $message, $context));
```

Comment:
Is 'system.' assumed here, or is that a sort of legacy deal no longer needed?

Reply:
Event registration was only bing registered to 'onLog' (instead of the more explicit 'system.onLog'), so I was making it consistent. It could go the other direction if we want to enforce a more specific event listener group (with this behavior, "onLog" on any loaded plugin (plg_members_*) could respond). That could be a good or bad thing depending on what you're doing.